qiskit_ibm_runtime/ibm_backend.py

```diff
@@ -675,6 +676,12 @@ def run(
                 - If ESP readout is used and the backend does not support this.
         """
         # pylint: disable=arguments-differ
+        issue_deprecation_msg(
+            msg="backend.run() is deprecated ",
+            version="0.23",
+            remedy="Please use the primitives instead.",
```

Comment:
We should link to the migration guide. @beckykd do you have the final url for the guide?

Reply:
Migration guide: https://docs.quantum.ibm.com/api/migration-guides/qiskit-runtime